modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -87,11 +85,11 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
             'Please submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.')
         end
 
-        ClaimsApi::Logger.log('validate_identifiers',
-                              rid: request.request_id, mpi_res_ok: mpi_add_response.ok?,
-                              ptcpnt_id: target_veteran.participant_id.present?,
-                              birls_id: target_veteran.birls_id.present?,
-                              icn: target_veteran&.mpi_icn)
+        claims_v1_logging('validate_identifiers', message: "multiple_ids: #{ids.size},
```

Comment:
Line 62 of the application controller has similar logging, should we switch to use `claims_v1_logging` there as well?